serving/installing-logging-metrics-traces.md

```diff
@@ -1,73 +1,87 @@
 # Monitoring, Logging and Tracing Installation
 
-Knative Serving offers two different monitoring setups: 
-One that uses Elasticsearch, Kibana, Prometheus and Grafana and 
-another that uses Stackdriver, Prometheus and Grafana. See below 
-for installation instructions for these two setups. You can install 
-only one of these two setups and side-by-side installation of these two are not supported.
+Knative Serving offers two different monitoring setups:
+[Elasticsearch, Kibana, Prometheus and Grafana](#Elasticsearch,-Kibana,-Prometheus-&-Grafana-Setup) or [Stackdriver, Prometheus and Grafana](#Stackdriver,-Prometheus-&-Grafana-Setup). You can install only one of these two setups and side-by-side installation of these two are not supported.
```

Comment:
Fold this (or the whole doc) at 80, if possible. One way to do this is with https://prettier.io and set --proseWrap=always